avParser/source/AVParser.cpp

```
@@ -95,11 +98,15 @@ namespace AVParser {
     currentFrame--;
   }
 
-  void MediaParser::loadFrameAt(const uint32_t targetFrame)
+  void MediaParser::loadFrameAt(uint32_t targetFrame)
   {
     if (targetFrame > getTotalFrames())
     {
-      throw std::out_of_range("Target frame is out of range!");
```

This function should throw a range error, instead users should be making sure to not send a frame beyond the max (since getTotalFrames is public)